python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -1439,6 +1439,444 @@ keys:
+                        include_sub_route_map:
+                          type: bool
+                          description: Include sub-route-map references in missing policy decision.
+                    direction_out:
+                      description: Missing policy outbound direction.
+                      type: dict
+                      keys:
+                        action:
+                          description: Missing policy action.
+                          type: str
+                          required: true
+                          valid_values:
+                            - deny
+                            - permit
+                            - deny-in-out
+                        include_community_list:
+                          type: bool
+                          description: Include community-list references in missing policy decision.
+                        include_prefix_list:
+                          type: bool
+                          description: Include prefix-list references in missing policy decision.
+                        include_sub_route_map:
+                          type: bool
+                          description: Include sub-route-map references in missing policy decision.
+                multi_path:
+                  type: bool
+                next_hop_resolution:
+                  type: dict
+                  keys:
+                    v4_mapped_v6_translation:
+                      type: bool
+                next_hop_self:
+                  type: bool
+                next_hop_self_source_interface:
+                  description: Source interface name.
+                  type: str
+                next_hop_self_v4_mapped_v6_source_interface:
+                  description: v4-mapped-v6 source interface name. Takes precedence over the next_hop_self_source_interface.
+                  type: str
+                next_hop_unchanged:
+                  type: bool
+                rcf_in:
+                  type: str
+                  description: |-
+                    Inbound RCF function name with parenthesis.
+                    Example: MyFunction(myarg).
+                rcf_out:
+                  type: str
+                  description: |-
+                    Outbound RCF function name with parenthesis.
+                    Example: MyFunction(myarg).
+                route_map_in:
+                  type: str
+                  description: Inbound route-map name.
+                route_map_out:
+                  type: str
+                  description: Outbound route-map name.
+          networks:
+            type: list
+            primary_key: prefix
+            min_length: 1
+            items:
+              type: dict
+              keys:
+                prefix:
+                  description: IPv4 prefix "A.B.C.D/E" or IPv6 prefix "A:B:C:D:E:F:G:H/I".
+                  type: str
+                route_map:
```

Comment:
Now we have option of rcf too.
```
wan1-dc1(config-router-bgp-af-label)#network 10.1.1.1/24 ?
  rcf        Routing control functions configuration to filter and modify paths
  route-map  Name a route map
  <cr>       
```

Reply:
RCF isn't supported in this address family - if you try, it puts the network statement under the IPv4 address family.
```
device...09:52:52(config-router-bgp-af-label)#network 22.22.33.0/24 rcf test()  
device...09:54:05(config-router-bgp)#show a | sec 33
router bgp 65101
   address-family ipv4
      network 22.22.33.0/24 rcf test()
```